metrics-operator/controllers/common/providers/elk/elk.go

```diff
@@ -0,0 +1,121 @@
+package elasticsearch
```

Comment:
please add this word int to list of expected words in the spelling workflow https://github.com/keptn/lifecycle-toolkit/blob/main/.github/actions/spelling/expect.txt